CMake/ITKModuleExternal.cmake

```diff
@@ -65,7 +65,7 @@ endif()
 
 if(ITK_WRAPPING)
   # WRAPPER_LIBRARY_OUTPUT_DIR. Directory in which generated cxx, xml, and idx files will be placed.
-  set(WRAPPER_LIBRARY_OUTPUT_DIR "${ITK_DIR}/Wrapping" CACHE INTERNAL "Need to specify the output library directory globally")
+  set(WRAPPER_LIBRARY_OUTPUT_DIR "${ITK_BINARY_DIR}/Wrapping" CACHE INTERNAL "Need to specify the output library directory globally")
```

Comment:
This should still be `ITK_DIR` for wrapping external modules. `ITK_BINARY_DIR` is only set when  building `ITK`.

Reply:
@thewtex  Thanks for catching this.  I did a test with an external build of AnisotropicDiffusionLBR with wrapping turned on.  It seemed to *not fail catastrophically* so I assumed the refactoring was OK.


I have removed these changes.